lib/record_store/provider/dnsimple.rb

```diff
@@ -9,11 +9,12 @@ def supports_alias?
 
       def add(record, zone)
         record_hash = api_hash(record, zone)
-        res = session.create_record(
+        res = session.zones.create_record(
+          account_id,
           zone,
-          record_hash.fetch(:name),
-          record.type,
-          record_hash.fetch(:content),
+          name: record_hash.fetch(:name),
+          type: record.type,
+          content: record_hash.fetch(:content),
           ttl: record_hash.fetch(:ttl),
           priority: record_hash.fetch(:priority, nil)
         )
```

Comment:
without looking deeper I can't tell whether it would be correct or not, but is this now just..

```
res = session.zones.create_record(
  account_id,
  zone,
  record_hash
)
```

?

Reply:
Here's the doc for `#create_record`: http://www.rubydoc.info/gems/dnsimple/Dnsimple/Client/ZonesRecords#create_record-instance_method

I guess that will do it.

Reply:
mostly I didn't know what would wind up in the record_hash... ie. is `type` included or not? are there additional attributes, or just these ones... 🤷‍♂️

Reply:
Yes. `type` is included in the `record_hash` right here: https://github.com/Shopify/record_store/pull/54/files#diff-edcaaf931e2c87e8e59ba598100e8095R128